JavaScript/JavaScriptSDK/Context/Session.ts

```diff
@@ -76,11 +79,14 @@ module Microsoft.ApplicationInsights.Context {
             // renew if acquisitionSpan or renewalSpan has ellapsed
             if (acquisitionExpired || renewalExpired) {
                 // update automaticSession so session state has correct id                
-                this.automaticSession.isFirst = undefined; 
+                this.automaticSession.isFirst = undefined;
                 this.renew();
             } else {
-                this.automaticSession.renewalDate = +new Date;
-                this.setCookie(this.automaticSession.id, this.automaticSession.acquisitionDate, this.automaticSession.renewalDate);
+                // do not update the cookie more often than cookieUpdateInterval
+                if (!this.cookieUpdatedTimestamp || now - this.cookieUpdatedTimestamp > _SessionManager.cookieUpdateInterval) {
+                    this.automaticSession.renewalDate = +new Date;
```

Comment:
Done. We have a [dataTime.Now()](https://github.com/Microsoft/ApplicationInsights-JS/blob/master/JavaScript/JavaScriptSDK/ajax/ajaxUtils.ts#L34) function which uses performance.now internally.